semgrep/sol-rules.t.sol

```diff
@@ -0,0 +1,403 @@
+// Semgrep tests for Solidity rules are defined in this file.
+// Semgrep tests do not need to be valid Solidity code but should be syntactically correct so that
+// Semgrep can parse them. You don't need to be able to *run* the code here but it should look like
+// the code that you expect to catch with the rule.
+
+contract SemgrepTest__sol_safety_deployutils_args {
+    function test() {
+        // ruleid: sol-safety-deployutils-args
+        DeployUtils.create1AndSave({
+            _save: this,
+            _name: "SuperchainConfig",
+            _args: abi.encodeCall(ISuperchainConfig.__constructor__, ())
+        });
+
+        // ruleid: sol-safety-deployutils-args
+        DeployUtils.create1({
+            _name: "SuperchainConfig",
+            _args: abi.encodeCall(ISuperchainConfig.__constructor__, ())
+        });
+
+        // ruleid: sol-safety-deployutils-args
+        DeployUtils.create2AndSave({
+            _save: this,
+            _salt: _implSalt(),
+            _name: "SuperchainConfig",
+            _args: abi.encodeCall(ISuperchainConfig.__constructor__, ())
+        });
+
+        // ruleid: sol-safety-deployutils-args
+        DeployUtils.create2({
+            _salt: _implSalt(),
+            _name: "SuperchainConfig",
+            _args: abi.encodeCall(ISuperchainConfig.__constructor__, ())
+        });
+
+        // ok: sol-safety-deployutils-args
```

Comment:
What’s the `ruleid` vs `ok` prefix? Would be helpful to add comments to the top explaining some of the semgrep testing basics here

Reply:
Added comment to the top of the file to explain